etc/helm/pachyderm/values.yaml

```diff
@@ -417,6 +417,7 @@ pachd:
     # object storage uploads per Pachd instance.  It is analogous to
     # the --upload-concurrency-limit argument to pachctl deploy.
     uploadConcurrencyLimit: 100
+  storageGCPolling: 1m
```

Comment:
Can we not have a default set? I would prefer to have defaults handled at the lowest level possible.